Zend/zend_builtin_functions.c

```diff
@@ -1009,7 +1009,7 @@ ZEND_FUNCTION(get_class)
 {
 	zval *obj = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o!", &obj) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o", &obj) == FAILURE) {
```

Comment:
oh, sorry. Right, I've missed that get_class() could be still a valid call. But it's now similar effect as get_called_class(), couldn't get_called_class() work in any methods and return current class name then get_class($object); would do only one thing, returns a class name of a given object and no more else :)